### PR TITLE
chore: add beliefs.social to ecosystem page

### PR DESCRIPTION
**Pull request by dalechyn**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by wbnns — CHANGES_REQUESTED** [review]
@dalechyn 

Hello, looks like a bunch of artifacts got brought into this PR. Can you please remove them so it's just the files you meant to change?

**Comment by dalechyn** [comment]
> @dalechyn 
> 
> 
> 
> Hello, looks like a bunch of artifacts got brought into this PR. Can you please remove them so it's just the files you meant to change?

Apologies, nvim linter linted the whole project.
Will do in the next 2 hrs.

**Comment by dalechyn** [comment]
Seems like the master branch was force-pushed back when I've added this PR.

**Review by wbnns — CHANGES_REQUESTED** [review]
@dalechyn 

Thanks for this PR! Could you please change the category to defi and also confirm if this is on Base or on Degen L3?

**Comment by dalechyn** [comment]
> @dalechyn
> 
> Thanks for this PR! Could you please change the category to defi and also confirm if this is on Base or on Degen L3?

Sure! Yes, this operates on base, you can check yourself at beliefs.social.

File: apps/web/src/data/ecosystem.json [hunk 1]
```diff
@@ -4223,5 +4223,14 @@
     "description": "The friendly crypto wallet built for DeFi & NFTs.",
     "url": "https://phantom.app/",
     "imageUrl": "/images/partners/phantom.svg"
+  },
+  {
+    "name": "Beliefs",
+    "tags": [
+      "social"
```
Comment (dalechyn) [inline apps/web/src/data/ecosystem.json:4230]:
```suggestion
      "defi"
```

**Comment by wbnns** [comment]
@dalechyn

Hi there! Apologies, we recently updated the way we [categorize and subcategorize apps](https://github.com/base-org/web?tab=readme-ov-file#updating-the-base-ecosystem-page); could you please fix the merge conflict and update the category + subcategory?

We'll do our best to review as soon as possible once you do! 🙏

**Comment by wbnns** [comment]
@dalechyn 

Please also rebase to resolve the merge conflict.

**Comment by wbnns** [comment]
@dalechyn Please open a new PR, when you are ready, incorporating the above changes. 🙏

**Pull request closed without merging**